pkgs/development/python-modules/pyro-api/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  version = "0.1.1";
+  pname = "pyro-api";
+
+  src = fetchPypi {
+    inherit version pname;
+    sha256 = "0rhd7p61pf2vvflbdixp7sygblvvl9qbqavxj27910lr79vl4fdz";
+  };
+
+  checkPhase = ''
+    python -c 'import pyroapi'
+  '';
```

Comment:
```suggestion
  pythonImportsCheck = [ "pyroapi" ];
```

Reply:
i tried this but it still runs the original `checkPhase` which attempts to run tests, which fail because they depend on `pyro-ppl` (which depend on this package). Not sure how this can be resolved

Reply:
you also want to do `doCheck = false;` and leave a comment as to why they are not enabled

Reply:
done